core/app.js

```diff
@@ -72,7 +74,7 @@ app.use(_.get("/message", function *() {
 app.use(_.get("/signin", function *(){
   ticket=this.request.query.ticket;
   var obj= yield auth.cas_login(ticket, SERVICE);
-  this.session.user=obj.username;
```

Comment:
Since @trevren11 and @alexcdahl are already expecting the `user` value to contain just the username, maybe it would be best to store the attributes separately, perhaps in `this.session.attributes`